### ISSUE TITLE
Loss parameters for downstream tasks

### ISSUE THREAD

**Issue by wenjiajia123** [showlab/UniVTG#21]
Hi, I got bad results when training on moment retraival without loading pretrained model, I wonder that are the hyperparameters in 'qvhl_pretrain.sh' correct? 🙂
`b_loss_coef=10
g_loss_coef=0
eos_coef=0.1
f_loss_coef=10
s_loss_intra_coef=0
s_loss_inter_coef=0`

**Comment by QinghongLin** [comment]
Hi, this setting is not the optimal result. Please refer our paper or checkpoint config for parameters. :)

**Comment by wenjiajia123** [comment]
Thamk u! I have found that. 😊
